open-budget/app/scripts/spendings_page.coffee

```diff
@@ -1,17 +1,27 @@
 class SpendingsPageView extends Backbone.View
 
-        initialize: ->
-            @model.on 'ready-spendings-page', => @render()
-            console.log 'SpendingsPageView init'
+    initialize: ->
+        @model.on 'ready-spendings-page', =>
+            @render()
+        console.log 'SpendingsPageView init'
 
-        render: ->
-            @$el.css('display','inherit')
-            data =
-                exemptions: _.map( @model.newSpendings.models, (x) -> x.toJSON() )
-            @$el.html window.JST.latest_spending_updates( data )
+    render: ->
+        @$el.css('display', 'inherit')
+        data =
+            exemptions: _.map(@model.newSpendings.models, (x) ->
+                x.toJSON())
+        @$el.html window.JST.latest_spending_updates(data)
 
-$( ->
-        console.log "spendings-page"
-        if window.pageModel.get("spendingsPage")?
-            window.spendingsPageView = new SpendingsPageView({el: $("#spendings-page-article .latest-updates"),model: window.pageModel});
+        # Initialize the on click event of the alerts
+        $("div.exemption-alert").on("click", (d) =>
```

Comment:
What would probably be better here is when that div is clicked, then update something in the model, like which spending was selected. 
 
Then, in another place, the view could subscribe to when the selected spending changed, and re-render itself or just update classes of the selected/deselected divs.   This way the source of the state comes from the models and the view just updates itself based on it.

We probably should add another field to the model about which spending is selected, but the challenge is that the spendings don't have an ID.  @akariv do you think it's possible to have the api return a spending id with each spending? Or are you the right one to ask?